### PR TITLE
datomic time travel initial

### PR DESCRIPTION
**Pull request by joelturnbull**
read it up! @mguterl @cdmwebs @mitchlloyd

### CODE REVIEW

**Comment by mguterl** [comment]
:thumbsup: I really like the title.

**Comment by cdmwebs** [comment]
This is really interesting, Joel. I'm really interested in things like this. I'm still not certain how it's different from couch.

**Comment by joelturnbull** [comment]
I was thinking this morning how I forgot to mention couch. Can you set couch back to a certain time and query it as if?

File: posts/2013-07-16-time-travel-your-data.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Simple example of a common problem. How do you store the value of a certain piece of data over time? What was the value of markup percentage in July? What was the status of this order three days ago? What did this property cost in 2011?
+
+Rails has extensions like [acts_as_audited](https://github.com/collectiveidea/acts_as_audited) that we've used with some success. Audit tables back model tables, storing attribute changes as data structures. And the idea of querying based on time is not new. Micheal Guterl pointed out to me the term "bitemporal databases". There are seemingly a few out there, and  a bitemporal package for Postgres.   
+
+My feeling is that Datomic might provide the easiest and most elegant solution to getting that type of information. In Diametric I watched Yoko pass a time object to the database and then query it in it's entirety as_of that moment in time. It's baked in. Data is always preserved, the past never changes, and time is always something you can query on.
+
+I'm a fan of writing about things I have no practical knowledge of. I'm interested in hearing thoughts and experiences on Datomic, Diametric, or temporal databases.
```
Comment (cdmwebs) [inline posts/2013-07-16-time-travel-your-data.md]: win.

**Pull request merged**